server/tinylicious/src/index.ts

```diff
@@ -20,8 +22,12 @@ import { TinyliciousRunnerFactory } from "./runnerFactory";
 // TODO: Set this globally since the Historian use the global Axios default instance.  Make this encapsulated.
 Axios.defaults.httpAgent = new http.Agent({ keepAlive: true });
 
+const configPath = path.join(__dirname, "../config.json");
+configureLogging(configPath);
+
```

Comment:
Newer `runService()` requires passing winston logger (and requires configuring it externally.)